bin/warnhist

```diff
@@ -212,17 +412,30 @@ while (<>) {
         $last_c_name = $_;
     }
 
+    # Compute LOC for compiled source files, if warning density requested
```

Comment:
Count lines-of-code for compiled files (lines that start with CC ..., FC ..., etc)